File: TiaCodegen/Commands/Comparisons/Gt.cs
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands.Comparisons
{
    public class Gt : CompareOperator // >
    {
        public Gt(params IOperationOrSignal[] operationOrSignals)
            : base(operationOrSignals)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/Comparisons/Gt.ts
import { IOperationOrSignal } from '../../Interfaces/IOperationOrSignal.js';
import { CompareOperator } from './CompareOperator.js';

export class Gt extends CompareOperator {
    constructor(...operationOrSignals: IOperationOrSignal[]) {
        super(...operationOrSignals);
    }
}
